src/constants/helmOptions.ts

```diff
@@ -0,0 +1,53 @@
+const helmCommonOptions = {
```

Comment:
could we add the description for each of these? or just have a link in the UI that takes the user to the Helm documentation?

Reply:
We can add these later, for now we don't have the design to include the description for each option